### ISSUE TITLE
FINN daily files for recent years

### ISSUE THREAD

**Issue by julietajuncosa** [geoschem/geos-chem#2442]
### Your name

Julieta Juncosa Calahorrano

### Your affiliation

UMN

### Please provide a clear and concise description of your question or discussion topic.

Hello, 

I am trying to run a GEOS-Chem simulation using the FINN inventory. However, I can only find the daily (monthly) files for up to 2016 in the UWash repository. I need to work with years from 2018 onwards. I tried to search on the NCAR webpage but the files listed there didn't seem to match the names in the UWash repositoty. Any information is welcome. 
Julieta

**Comment by yantosca** [comment]
Thanks for writing @julietajuncosa.  The FINN inventory hasn't really been used much in GEOS-Chem, it's always been a bit experimental.

The FINN data page is here:

- https://www2.acom.ucar.edu/modeling/finn-fire-inventory-ncar\

There seems to be an updated version from Christine Wiedinmyer as of 2023, see:

- https://gmd.copernicus.org/articles/16/3873/2023/

From the paper:

>Data availability
>Emissions calculated from the FINNv2.5 algorithms, for MODIS and MODIS + VIIRS fire detections with MOZART VOC speciation types, are archived at https://doi.org/10.5065/XNPA-AF09 (Wiedinmyer and Emmons, 2022) and https://doi.org/10.5281/zenodo.7868652 (Wiedinmyer and Emmons, 2023). The FINNv2.5 emissions, including emissions files for SAPRC and GEOS-Chem VOC speciation, in addition to tools for subsetting, are available from the NCAR Research Data Archive (https://rda.ucar.edu/datasets/ds312.9/, last access: 7 July 2023; https://doi.org/10.5065/XNPA-AF09; Wiedinmyer and Emmons, 2022). Files are provided with emissions for each fire (ungridded; zipped text files) and are also gridded to 0.1×0.1∘ (netCDF files). Both sets of files are available for emissions from MODIS-only and MODIS + VIIRS fire detections at https://doi.org/10.5065/XNPA-AF09 (Wiedinmyer and Emmons, 2022).

I hope this helps.  Maybe questions are best directed to Christine Wiedinmyer (a link to her email is in the 2023 article).

**Comment by yantosca** [comment]
Also tagging the Emissions WG co-chairs: @eamarais @jaegle

**Comment by yantosca** [comment]
@julietajuncosa: you might need to update the HEMCO_Config.rc file for the newer FINN data.  I think the version we had was 1.5 but they are on version 2.5. now.  If they have gridded emissions then you may not need to use the FINN extension, you can treat it like GFAS (i.e. read it in as as a base emissions with extension # 0).

**Comment by eamarais** [comment]
Hi @julietajuncosa, I don't have anything more to add, as Bob has already helpfully provided links to everything you need to proceed if you wanted to incorporate the latest version of FINN in GEOS-Chem. There are also other biomass burning inventories to choose from, unless you were specifically interested in FINN for a particular purpose.

**Comment by yantosca** [comment]
Closing out this issue.

**Comment by eamarais** [comment]
Hi @Hemrajbhattarai, I actually conducted a project at the start of the year that involved embedding FINN v2.5 in GEOS-Chem. Happy to share my HEMCO_Config and files with you. My implementation was for 2018 and 2019, but I didn't have to change the original FINNv2.5 files to get it to work. 

@yantosca, the input files could also be included in the GEOS-Chem data repositories for wider use of the FINN2.5 inventory. Perhaps easiest if I share the bash scripts I used to download all the files I included in GEOS-Chem?
